WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInActivity.java

```diff
@@ -195,12 +213,16 @@ public SmartLockHelper getSmartLockHelper() {
     }
 
     private void popBackStackToSignInFragment() {
-        FragmentManager fragmentManager = getSupportFragmentManager();
-        while (fragmentManager.getBackStackEntryCount() > 1) {
-            fragmentManager.popBackStackImmediate();
-        }
 
-        getSupportFragmentManager().popBackStack();
+        // this can bee called asynchronously from Volley onError handler so we must check if state is OK
```

Comment:
Volley's error callback happens on the main thread, so I don't think this is the problem. I'm not sure, but perhaps the issue is [here](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/5661-magic-link-crash/WordPress/src/main/java/org/wordpress/android/ui/accounts/login/MagicLinkRequestFragment.java#L162), where we call `mListener.onEnterPasswordRequested()` without checking `isAdded()`?